openbas-front/src/components/Autocomplete.js

```diff
@@ -56,7 +73,8 @@ const renderAutocomplete = ({
         <IconButton
           onClick={() => openCreate()}
           edge="end"
-          style={{ position: 'absolute', top, right: 35 }}
+          className={injectorContractUpdate ? classes.iconInjector : classes.iconDefault}
```

Comment:
Is not a good practice to add a specific variable like this in a generic component

Reply:
And the add and delete icons overlap
![image](https://github.com/OpenBAS-Platform/openbas/assets/53513584/e8c10184-4783-452b-91b4-e63fb0522c78)